packages/builders/scripts/testing/recorded-retired-instances.js

```diff
@@ -26,8 +26,7 @@ export const testRecordedRetiredInstances = async ({
   assert(auctionIDs.length === 1);
   const auctionInstance = retiredContractInstances.get(auctionIDs[0]);
   trace({ auctionInstance });
-  // I don't know why it's neither in governedContractKits nor contractKits
-  // assert(await E(governedContractKits).get(auctionInstance));
+  assert(await E(governedContractKits).get(auctionInstance));
```

Comment:
glad we had this test ready to go, let's figure out later why it got commented out

Reply:
I commented it out, because I thought the important thing was saving the instance.